internal/cluster/distribution/eks/eksprovider/workflow/create_node_pool_workflow_test.go

```diff
@@ -747,6 +747,23 @@ func TestCreateNodePoolWorkflowExecute(t *testing.T) {
 				},
 			)
 
+			environment.RegisterActivityWithOptions(
+				func(
+					ctx context.Context,
+					input CalculateNodePoolVersionActivityInput,
+				) (*CalculateNodePoolVersionActivityOutput, error) {
+					if testCase.expectedError != nil &&
+						strings.HasPrefix(testCase.expectedError.Error(), "node pool calculation error") {
+						return nil, testCase.expectedError
+					}
+
+					return &CalculateNodePoolVersionActivityOutput{}, nil
```

Comment:
**Required**: we should return a fake value such as `test-node-pool-version` here for easier test debugging.